shotgun_api3/lib/mimetypes.py

```diff
@@ -31,6 +31,7 @@
 import sys
 import posixpath
 import urllib
+import urllib.parse
```

Comment:
urllib.splittype no longer exists in python3. It's been moved to the parse module.